### PR TITLE
Add new ComboBox features and update articles.

### PR DESCRIPTION
**Pull request by mariaterzieva12**

### CODE REVIEW

File: controls/chat/item-template-selector.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ The following example demonstrates how to create a custom `ChatItemTemplateSelec
 
 The image below shows how the customized Chat control could look:
 
-![Chat Customization](images/chat-customization.png)
+![Chat ](images/chat-item-template-selector.png)
```
Comment (yordan-mitev) [inline controls/chat/item-template-selector.md:64]:
```suggestion
![A customized .NET MAUI Chat](images/chat-item-template-selector.png)
```

File: controls/slideview/styling/indicator-styling.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Add Style resources:
 
 See the result below:
 
-![SlideView IndicatorStyling](images/slideview-indicatorstyling.png)
+![SlideView Indicator Styling](images/slideview-indicator-styling.gif)
```
Comment (yordan-mitev) [inline controls/slideview/styling/indicator-styling.md:45]:
```suggestion
![.NET MAUI SlideView Indicator Styling](images/slideview-indicator-styling.gif)
```

File: controls/slideview/styling/navigation-buttons-styling.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ Add Style resources:
 
 And the result below:
 
-![SlideView IndicatorStyling](images/slideview-navigation.png)
+![SlideView Navigation Buttons Styling](images/slideview-navigation-buutons-styling.gif)
```
Comment (yordan-mitev) [inline controls/slideview/styling/navigation-buttons-styling.md:29]:
```suggestion
![.NET MAUI SlideView Navigation Buttons Styling](images/slideview-navigation-buutons-styling.gif)
```

File: controls/combobox/commands.md [hunk 4]
```diff
@@ -14,7 +14,15 @@ ComboBox has the following commands:
 
 > SelectAll command can be used only when `SelectionMode` is `Multiple`. An exception will be thrown, if the command is invoked in `Single` `SelectionMode`.
```
Comment (yordan-mitev) [inline controls/combobox/commands.md:15]:
```suggestion
> The `SelectAll` command can be used only when `SelectionMode` is `Multiple`. An exception will be thrown if the command is invoked in `Single` `SelectionMode`.
```

File: controls/combobox/commands.md [hunk 5]
```diff
@@ -14,7 +14,15 @@ ComboBox has the following commands:
...
 
-- `ClearSelectionCommand` (*ICommand*)&mdash;Sets the selection to null. If Multiple SelectionMode is used, this command will clear all selected items.
+- `ClearSelectionCommand` (`ICommand`)&mdash;Sets the selection to null. If Multiple SelectionMode is used, this command will clear all selected items.
```
Comment (yordan-mitev) [inline controls/combobox/commands.md:17]: `SelectionMode` or simply "selection mode"
```suggestion
- `ClearSelectionCommand` (`ICommand`)&mdash;Sets the selection to null. If Multiple `SelectionMode` is used, this command will clear all selected items.
```

File: controls/combobox/configuration.md [hunk 6]
```diff
@@ -23,13 +23,21 @@ and the result:
 
 ![ComboBox Placeholder](images/combobox-visual-structure.png)
 
+Here is an example of setting the `PlaceholderColor` property:
+
+<snippet id='combobox-configuration-placeholder-color'/>
+
+and the result:
```
Comment (yordan-mitev) [inline controls/combobox/configuration.md:30]: Use complete sentences.

File: controls/combobox/configuration.md [hunk 7]
```diff
@@ -23,13 +23,21 @@ and the result:
...
+
+![ComboBox Placeholder Color](images/combobox-placeholder-color.png)
```
Comment (yordan-mitev) [inline controls/combobox/configuration.md:32]:
```suggestion
![.NET MAUI ComboBox Placeholder Color](images/combobox-placeholder-color.png)
```

File: controls/combobox/configuration.md [hunk 8]
```diff
@@ -47,9 +55,22 @@ To manage the dropdown of the ComboBox, use the following properties:
 
 * `DropDownWidth`(`double`)&mdash;Defines the width of the dropdown of the control.
 * `DropDownHeight`(`double`)&mdash;Defines the height of the dropdown of the control.
+* `DropDownVerticalOffset`(`double`)&mdash;Defines the vertical offset of the drop down part of the control. This property allows an option to modify the control with no space between the ComboBox and the dropdown.
 * `IsDropDownOpen`(`bool`)&mdash;Defines whether the dropdown part of the control is opened. The default value is `true`. 
 * `IsDropdownClosedOnSelection`(`bool`)&mdash;Defines whether the dropdown will be closed when the item is selected or deselected. The default value is `true`.
 
+Here is an example with `DropDownWidth` property set:
```
Comment (yordan-mitev) [inline controls/combobox/configuration.md:62]:
```suggestion
Here is an example that uses the `DropDownWidth` property:
```

File: controls/combobox/configuration.md [hunk 9]
```diff
@@ -47,9 +55,22 @@ To manage the dropdown of the ComboBox, use the following properties:
...
+
+<snippet id='combobox-configuration-dropdownwidth'/>
+
+Here is an example with `DropDownHeight` property set:
```
Comment (yordan-mitev) [inline controls/combobox/configuration.md:66]:
```suggestion
Here is an example that uses the `DropDownHeight` property:
```

File: controls/combobox/configuration.md [hunk 10]
```diff
@@ -47,9 +55,22 @@ To manage the dropdown of the ComboBox, use the following properties:
...
+
+<snippet id='combobox-configuration-dropdownheight'/>
+
+Here is an example with `DropDownVerticalOffset` property set:
```
Comment (yordan-mitev) [inline controls/combobox/configuration.md:70]:
```suggestion
Here is an example that uses the `DropDownVerticalOffset` property:
```

**Review by yordan-mitev — APPROVED** [review]

File: controls/combobox/commands.md [hunk 11]
```diff
@@ -12,9 +12,17 @@ ComboBox has the following commands:
 
 - `SelectAllCommand` (`ICommand`)&mdash;Selects all items from the source.
 
-> SelectAll command can be used only when `SelectionMode` is `Multiple`. An exception will be thrown, if the command is invoked in `Single` `SelectionMode`.
+> The `SelectAll` command can be used only when `SelectionMode` is `Multiple`. An exception will be thrown if the command is invoked in `Single` `SelectionMode`.
 
-- `ClearSelectionCommand` (*ICommand*)&mdash;Sets the selection to null. If Multiple SelectionMode is used, this command will clear all selected items.
+- `ClearSelectionCommand` (`ICommand`)&mdash;Sets the selection to null. If Multiple `SelectionMode` is used, this command will clear all selected items.
+
+Here is an example with `ClearSelectionCommand`:
```
Comment (didiyordanova) [inline controls/combobox/commands.md:19]: Add more information that you can override the default behavior and create custom command, like in the example. Also explain that the example shows both states, default execution and custom command execution

**Review by didiyordanova — APPROVED** [review]

**Pull request merged**